uploading homework 12

edisabou opened the pull request.

edisabou committed:
uploading homework 12

mariuspodean commented on lesson_12/less12.py:
```diff
@@ -0,0 +1,55 @@
+        del self._crayons[key]
+
+    def insert(self, index, item):
+        return self._crayons[index][item]
```
Please add a test for this method also; are you sure this works?

edisabou replied:
changed the return for `insert`

also: reuploaded, because of folder structure
do not know how to remove the initial commit

edisabou replied:
thought I removed this initial commit with my last commit, but no. let me know how I can fix this. thanks.

edisabou committed:
re-uploading - new folder structure

edisabou committed:
changed the return for insert
also: reuploaded, because of folder structure
do not know how to remove the initial commit

edisabou committed:
test: think I managed to remove the initial push removed
